### ISSUE TITLE
failed to resolve: use of undeclared crate or module `anyhow`

### ISSUE THREAD

**Issue by stevenliebregt** [dimo414/parameterized_test#1]
After updating from version `0.1.0` to `0.2.0` the following error appears

**Comment by dimo414** [comment]
Thanks for the report, and sorry for the trouble. Can you help me troubleshoot further? An example reproduction or more details would be helpful.

For context, there is an [optional dependency on `anyhow`](https://github.com/dimo414/parameterized_test/blob/e19549cf6e27b1d0534f9d3cfe043940064adacd/Cargo.toml#L22) now, so at a minimum you should be able to work around this issue by disabling the `propagation` feature (which is enabled by default).

**Comment by dimo414** [comment]
Ok, I was able to reproduce this in an empty Cargo project:

```
$ cargo test
    Updating crates.io index
   Compiling anyhow v1.0.66
   Compiling parameterized_test v0.2.0
   Compiling foo v0.1.0 (/private/tmp/foo)
error[E0433]: failed to resolve: use of undeclared crate or module `anyhow`
  --> src/main.rs:8:5
   |
8  | /     even! {
9  | |         bad_case:  1, // this test case will fail
10 | |         good_case: 2,
11 | |     }
   | |_____^ use of undeclared crate or module `anyhow`
   |
   = note: this error originates in the macro `even` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0433`.
error: could not compile `foo` due to 2 previous errors
```

Looks like I'm just [missing a `use` declaration](https://users.rust-lang.org/t/proc-macros-using-third-party-crate/42465/4); let me try to fix this.

**Comment by dimo414** [comment]
@stevenliebregt would you mind trying 49656a6c and confirming if it resolves your error? If so I'll publish this as 0.2.1 and [yank](https://doc.rust-lang.org/cargo/reference/publishing.html#cargo-yank) 0.2.0.

**Comment by stevenliebregt** [comment]
@dimo414 Tested it out and it works fine now!

**Comment by dimo414** [comment]
Published 0.2.1 and yanked 0.2.0. Thanks again for the report!
